src/Datadog.Trace/Conventions/OtelOutboundHttpConvention.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.Conventions
+{
+    internal class OtelOutboundHttpConvention : IOutboundHttpConvention
+    {
+        private readonly Tracer _tracer;
+
+        public OtelOutboundHttpConvention(Tracer tracer)
+        {
+            _tracer = tracer;
+        }
+
+        public Scope CreateScope(OutboundHttpArgs args, out HttpTags tags)
+        {
+            var otelTags = new OtelHttpTags();
+            tags = otelTags;
+
+            string operationName = "HTTP " + args.HttpMethod;
+            string serviceName = _tracer.Settings.GetServiceName(_tracer, "http-client");
+            var scope = _tracer.StartActiveWithTags(operationName, tags: tags, serviceName: serviceName, spanId: args.SpanId);
+            scope.Span.Type = SpanTypes.Http;
+
+            var uri = args.RequestUri;
+            otelTags.HttpMethod = args.HttpMethod;
+            otelTags.HttpUrl = string.Concat(uri.Scheme, Uri.SchemeDelimiter, uri.Authority, uri.PathAndQuery, uri.Fragment);
+            otelTags.HttpScheme = uri.Scheme;
+            otelTags.HttpHost = uri.Authority;
+            otelTags.HttpTarget = uri.PathAndQuery + uri.Fragment;
```

Comment:
TBH I am not sure if we need (and want) these.
Reference: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md

Reply:
Well these are not `Required` by the specs, maybe we can think later if we should have an optional flag setting to expose those keys, but by default only use the required one for performance reasons???

Reply:
Sounds good to me

Reply:
@pellared please, create an issue to add an option for the extra tags.